src/app/core/market/api/template/image/image.model.ts

```diff
@@ -0,0 +1,27 @@
+
+export class Image {
+  constructor(private image: any) {
+  }
+
+  get thumbnail(): string {
+    const image = this.image.ItemImageDatas.find((o) => o.imageVersion === 'THUMBNAIL')
+    return image.dataId ? image.dataId : './assets/images/placeholder_1-1.jpg'
```

Comment:
Check if this works:
`return image.dataId || './assets/images/placeholder_1-1.jpg'`

Reply:
yes it does :)